presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -99,6 +99,11 @@ String
 
     Example type definitions: ``varchar``, ``varchar(20)``
 
+    SQL statements support usage of Unicode literals with the prefix ``U&``,
```

Comment:
Let's change this a bit to use multiple examples. The example isn't quite right. I'm looking at `io.prestosql.sql.parser.TestSqlParser#testUnicodeString` for the correct syntax.
```
String values can be written as simple literals, or Unicode escape syntax with the ``U&`` prefix.
The escape character defaults to ``\``.

Examples:
* ``hello world``
* ``U&'hello\1F600world'``
* ``U&'hello#1F600world' UESCAPE '#'``

Reply:
Oh ... thats different from what I assumed..

Reply:
Adapted to that syntax now.